eclipse-theia/theia
# Added tab related APIs - without functionality implemented yet

#### What it does
It adds tab related APIs. For now it just solves the problem that we get an error when an extension uses the VSCode Tab API.
Functionality is not implemented on theia side yet. 

#### How to test

1. Download this test extension: 
[tab-test-0.0.1.zip](https://github.com/eclipse-theia/theia/files/10318724/tab-test-0.0.1.zip)
and change the file type from zip to vsix
or create it [here](https://github.com/jbicker/tab-api-test-extension)
or use another extension which uses the VSCode Tab API
2. install it in theia
3. trigger the command 'Close Active Tab'
4. see that nothing happens, even no errors notified anymore

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
